src/views/sidebar/tests/edit-modal.test.tsx

```diff
@@ -342,4 +342,121 @@ describe('edit-modal', () => {
 		expect(action.color).toBe(folder?.color ?? 0);
 		expect(action.name).toBe(folderName);
 	});
+
+	test('folder name disable when edit system folder', async () => {
+		const closeModal = jest.fn();
+		const store = generateStore();
+		const folder: Folder = {
+			id: FOLDERS.INBOX,
+			uuid: faker.datatype.uuid(),
+			name: 'folders.inbox',
+			absFolderPath: '/Inbox',
+			l: FOLDERS.USER_ROOT,
+			luuid: faker.datatype.uuid(),
+			checked: false,
+			f: 'ui',
+			u: 37,
+			rev: 1,
+			ms: 2633,
+			n: 889,
+			s: 174031840,
+			i4ms: 33663,
+			i4next: 17222,
+			activesyncdisabled: false,
+			webOfflineSyncDays: 30,
+			recursive: false,
+			deletable: false,
+			isLink: false,
+			children: [],
+			parent: undefined,
+			depth: 1
+		};
+
+		const { user } = setupTest(<EditModal onClose={(): void => closeModal()} folder={folder} />, {
+			store
+		});
+
+		expect(screen.getByTestId('folder-name')).toBeInTheDocument();
+		const newFolder = screen.getByTestId('folder-name');
+		const folderInputElement = within(newFolder).getByRole('textbox');
+		expect(folderInputElement).toBeDisabled();
```

Comment:
I think this is the only assertion that is needed. All the other assertion are already covered by other existing tests